docs/source/usersguide/input.rst

```diff
@@ -811,7 +790,7 @@ OpenMOC has an integrated CMFD acceleration framework that allows users to great
     cmfd.setOpticallyThick(False)
     cmfd.setSORRelaxationFactor(1.5)
     cmfd.setMOCRelaxationFactor(0.66)
```

Comment:
I also removed a full paragraph description of `Cell:setMOCRelaxationFactor()` below.